cli/flags.go

```diff
@@ -595,6 +595,13 @@ var (
 			Usage: "name of the imagecatalog",
 		},
 	}
+	FlImageCatalogOptional = StringFlag{
```

Comment:
On master a change has been merged which introduced the same flag: FlImageCatalog Can you please either use that one or rename it to this one and use the same everywhere. Either way is fine by me.